---
layout: page
permalink: sessions/session_8/practical
menubar_toc: true
---

<script link="{{ site.baseurl }}/assets/js/vanilla-back-to-top.min.js"></script>
<script>addBackToTop()</script
<script src="{{ site.baseurl }}/assets/js/copyCodeSnippet.js" defer></script>
<script src="{{ site.baseurl }}/assets/js/copyCodeBlock.js" defer></script>

## Practical session 

This workshop will explore the essential steps for detecting mosaic chromosomal alterations (mCAs).

## Practical Overview:
- Convert GenomeStudio CNV table to VCF
- Perform Phasing using [EAGLE2](https://alkesgroup.broadinstitute.org/Eagle/)
- Detect mCAs using [MoChA](https://github.com/freeseek/mocha) on the phased VCF
- plot the identified mCAs
- Utilize [Circos](http://circos.ca/) to illustrating the distribution of autosomal mCAs categorized as Gain, CNLOH, and Loss

### Let's get started 
Please go ahead and open the [08_mCA_colab.ipynb](https://github.com/DCEG-workshops/statgen_workshop_tutorial/blob/main/src/08_mCA_colab.ipynb) Google colab notebook. 
