---
layout: page
permalink: sessions/session_3/practical
menubar_toc: true
---

<script link="{{ site.baseurl }}/assets/js/vanilla-back-to-top.min.js"></script>
<script>addBackToTop()</script
<script src="{{ site.baseurl }}/assets/js/copyCodeSnippet.js" defer></script>
<script src="{{ site.baseurl }}/assets/js/copyCodeBlock.js" defer></script>

## Practical session 

This workshop deploys a precoded Jupyter notebook through Google Colaboratory to perform Fine mapping and colocalization analyses . We will also explore data visualization of the results. The majority of the session (except for mounting and setup) will be performed in R.

## Objectives

- To perform fine-mapping analysis, with only one potential causal variant in the region
- Visualize the results
- To perform colocalization of a GWAS significant loci with cis-eQTL data from two tissues
- To perform fine-mapping analysis, with more than one potential causal variants in the region
- Visualize the results


## Prerequisites

Before starting this workshop, we recommend that you have:

- A Google Account, completed Workshop #1 and been introduced to the usage of Google Colab, PLINK software, and data QC.
- A basic understanding of the pipeline of GWAS analysis and expected output from genome-wide association studies (GWAS) (covered in Workshop #2)
- Basic R programming for visualization.

